installer/util.sh

```diff
@@ -19,7 +19,7 @@ PROJECT_NAME='SIM'
 LOG_DIR=/var/log/sodafoundation
 LOGFILE=${LOGFILE:-/var/log/sodafoundation/sim_pre_installer.log}
```

Comment:
please change all reference to sim as delfin

Reply:
Done